docs/training/lerobot_guide.rst

```diff
@@ -359,6 +359,127 @@ Instructions for downloading and using these datasets can be found at the follow
 
 `Trossen Robotics Community <https://huggingface.co/TrossenRoboticsCommunity>`_
 
+
+
+===============================
+LeRobot X Aloha Solo User Guide
+===============================
+
+The process for using Aloha Solo is similar to Aloha Stationary. However, there are key differences in the configurations, policies, and environment settings that must be applied. This guide outlines the specific changes needed to set up and run Aloha Solo.
+
+Key Differences Between Aloha Solo and Aloha Stationary
+=======================================================
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+
+   * - **Feature**
+     - **Aloha Stationary**
+     - **Aloha Solo**
+   * - **Robot Configuration**
+     - ``aloha_stationary.yaml``
+     - ``aloha_solo.yaml``
+   * - **Policy**
+     - ``act_aloha_real``
+     - ``act_aloha_solo_real``
+   * - **Environment**
+     - ``aloha_real``
+     - ``aloha_solo_real``
+   * - **Camera Support**
+     - ``cam_high``, ``cam_low``, ``cam_left_wrist``, ``cam_right_wrist``
+     - ``cam_high`` and either ``cam_left_wrist`` or ``cam_right_wrist``
+
+
+Steps to Configure Aloha Solo
+=============================
+
+#. **Place the Required Files**:
```

Comment:
We don't bold the enumerated steps above (or anywhere else I think?). Be consistent with formatting